### DISCUSSION TITLE
Cron jobs don't run after server restart

### DISCUSSION THREAD

**Discussion by hansblaauw64** [Budibase/budibase#12694] [Help]
Hi,

We are restarting our server every night. We noticed that cronjob automations don't seem to run after each restart.

Is there a way to check what is going wrong?

**Comment by melohagan** [comment]
Hey @hansblaauw64 

Could you share some screenshots of your automations (trigger and steps) please?

**Reply by hansblaauw64** [reply]
![cronjob](https://github.com/Budibase/budibase/assets/39296077/cb9872ca-f9cf-44b1-90dd-6f1ffb7cb112)

Hi, it's a very simple cronjob with 1-2 steps. It never gets executed.

Maybe after a restart we need to get the cronjobs going somewhere?

We use self hosted.

**Reply by melohagan** [reply]
I wonder if you just need to republish your app after restarting

**Comment by hansblaauw64** [comment]
Hi,

For other functionality we don't need to re-publish. That would be a bummer
but could be solved.

Op vr 5 jan 2024 om 11:25 schreef melohagan ***@***.***>:

> I wonder if you just need to republish your app after restarting
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/12694#discussioncomment-8021627>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLZYTKRW73H5WWHS4R4SX3YM7IKJAVCNFSM6AAAAABBLGMB7SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAMRRGYZDO>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Hans Blaauw

*E: ***@***.***
*W:* http://www.bluenow.nl
*T:* +31(0)6 430 44 656

**Reply by melohagan** [reply]
Well I'm not sure if this is the case, just a thought. Would you be able to report back if you find that publishing the app after server restarts does also restart the cron jobs please?
